src/applications/caregivers/components/AdditionalInfo/index.jsx

```diff
@@ -1,27 +1,120 @@
-import React from 'react';
+import React, { useState } from 'react';
 import PropTypes from 'prop-types';
 import AdditionalInfo from '@department-of-veterans-affairs/formation-react/AdditionalInfo';
 import { links } from 'applications/caregivers/definitions/content';
+import Modal from '@department-of-veterans-affairs/formation-react/Modal';
 
-export const VetInfo = () => (
-  <p>
-    Complete the following information about the Veteran or Service member. You
-    aren’t required to fill in all fields, but we can review your application
-    faster if you provide more information.
-  </p>
+export const VetInfo = ({ pageTitle }) => (
+  <>
+    {pageTitle && <h5 className="vads-u-font-size--h4">{pageTitle}</h5>}
+    <p className="vads-u-margin-top--2">
+      Please complete all of the following information
+    </p>
+  </>
 );
 
-export const PrimaryCaregiverInfo = ({ additionalInfo }) => (
-  <div>
-    <p>
-      Complete the following information about the Caregiver or Family member
-      who is helping the Veteran or Service member. You can also include
-      information about up to two Secondary Family Caregivers.
+VetInfo.propTypes = {
+  pageTitle: PropTypes.string,
+};
+
+export const LearnMoreFamilyCaregiver = () => {
+  const [isModalVisible, toggleModal] = useState(false);
+
+  return (
+    <>
+      <Modal
+        title="Primary Family Caregiver"
+        id="primary-family-caregiver-info"
+        cssClass="caregiver-modal"
+        visible={isModalVisible}
+        onClose={() => toggleModal(false)}
+      >
+        <>
+          <p>
+            For the purpose of the program, Primary Family Caregivers are people
+            designated as a primary provider of personal care services to a
+            Veteran. They can be the Veteran’s or service member’s:
+          </p>
+
+          <ul>
+            <li>Parent</li>
+            <li>Spouse</li>
+            <li>Son or daughter</li>
+            <li>Step-family member</li>
+            <li>Extended family member</li>
+            <li>
+              Someone who is not a family member but resides with the Veteran or
+              will do so upon approval
+            </li>
+          </ul>
+        </>
+      </Modal>
+
+      <button
+        type="button"
+        className="va-button-link"
+        onClick={() => toggleModal(!isModalVisible)}
+      >
+        Learn more about what this means
```

Comment:
I know this is in the design, but I don't think this text is good for accessibility. It's weird that it's a modal inside a dropdown. Has @jenstrickland seen this? ([design page](https://vsateams.invisionapp.com/share/9XWSQZ6SMG5#/screens/413997345)).

Reply:
Thanks, @Mottie, I have not seen this. @ala-yna @sshein @jonathanjnelson can you tell me more about the decisions here? I suspect there are objectives here I am unaware of.

- modal inside of a dropdown — will introduce usability and accessibility challenges, as well as complicate QA
- link reads "Learn more about what this means" (for WCAG, the link should read Learn more about Primary Family Caregivers)

Is this a remnant of the challenge between legalese and plain language? From a 508 lens, the content in the modal **should** be the content in the additional info show/hide, and the legalese (if required here) would be at the bottom of it. If a case needs to be made with stakeholders, here are a couple of references for reasoning:

- [W3C WCAG 2.4.4 Link Purpose](https://www.w3.org/TR/WCAG20-TECHS/F63.html)

- [18f Plain Language, when to use legal and technical terms](https://content-guide.18f.gov/plain-language/#when-to-use-legal-and-technical-terms)

I hope this helps, and if I'm missing context, I'm all 👂 👂 .

Reply:
Hi @jenstrickland! You're right, this is due to the feedback from our stakeholders regarding legalese. Our other option would be to remove the plain language all together, which seemed like a worse user experience. This is a temporary fix, designed to meet our stakeholder requirements and give the Veterans a more friendly explanation to work with. We hope to move past the modal solution once we're able to build a better relationship with the Program team. In the meantime, we will certainly be presenting the stakeholder team with this feedback during our upcoming Staging review.

Reply:
Ah, I suspected as much, @ala-yna. Would it help for the stakeholders to learn Section 508 and the [Plain Writing Act of 2010](https://plainlanguage.gov/guidelines/) require this?